srml/grandpa/src/lib.rs

```diff
@@ -411,8 +420,8 @@ impl<FullIdentification: Clone> Offence<FullIdentification> for GrandpaEquivocat
 		self.validator_set_count
 	}
 
-	fn time_slot(&self) -> TimeSlot {
-		self.round as TimeSlot
+	fn time_slot(&self) -> GrandpaTimeSlot {
```

Comment:
I rather prefer `Self::TimeSlot` - less alterations if we change the type or rename it.